web/static/js/calendar.js

```diff
@@ -0,0 +1,318 @@
+
+  const days = [
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday"
+  ];
+
+  function isoDate ( date ) {
+    return date.toISOString( ).slice( 0, 10 );
+  }
+
+  function getMonday ( currentDate ) {
+    const now = currentDate ? new Date( currentDate ) : new Date();
+    const day = now.getDay( );
+    const diff = now.getDate( ) - day + ( day == 0 ? -6 : 1 );
+
+    const monday = new Date( now.setDate( diff ) );
+    return isoDate( monday );
+  }
+
+  function toHours ( timeString ) {
+    return timeString.split( /[:.]/ ).map(
+      ( value, index ) => value * Math.pow( 60, -index )
+    ).reduce(
+      ( a, b ) => a + b
+    );
+  }
+
+  function renderName ( user ) {
+    return [
+      user.name,
+      user.family_name,
+      user.nick_name && `(${user.nick_name})`
+    ].filter(
+      ( item ) => !!item
+    ).join( " " );
+  }
+
+  function renderDay ( date ) {
+    return days[ date.getDay()-1 ] || "";
+  }
+
+  function renderHour ( hour ) {
+    return `0${hour}:00`.slice( -5 );
+  }
+
+  function renderCalendar ( startDate, classes, calendar ) {
+    const monday = new Date( startDate );
+
+    let days = new Array( 5 ).fill( monday ).map(
```

Comment:
Does that mean we are displaying from Monday to Friday? I think we should include the whole week, the weekend is the most important for OLIve if I am not wrong.

@ghatighorias is my assumption correct?